internal/scan-manager/common/common/src/main/java/org/wso2/security/tools/scanmanager/common/util/HTTPUtil.java

```diff
@@ -64,10 +85,121 @@
      * @return a response entity object
      * @throws RestClientException when an error occurs while initiating the request
      */
-    public static ResponseEntity<String> sendDelete(HTTPRequest httpRequest) throws RestClientException {
+    public static ResponseEntity<String> sendDELETE(HTTPRequest httpRequest) throws RestClientException {
```

Comment:
IMO, `sendDelete` is the correct method name here. Please create a issue to correct this commonly across scanner in future.  (for other methods too)

Reply:
Created issue #131 to track this.